HelpSource/Classes/ServerOptions.schelp

```diff
@@ -140,6 +140,11 @@ A Boolean indicating whether the server should try to lock its memory into physi
 method:: maxLogins
 An Integer indicating the maximum number of clients which can simultaneously receive notifications from the server. When using TCP this is also the maximum number of simultaneous connections. This is also used by the language to split ranges of link::Classes/Node##Nodes::, link::Classes/Buffer##Buffers::, or link::Classes/Bus##Busses::. In multi-client situations you will need to set this to at least the number of clients you wish to allow. This must be the same in the Server instances on every client. The default is 1.
 
+method:: safetyClipThreshold
+note:: MacOS only::
+A Float indicating a safety threshold for output values to be clipped. This is necessary on MacOSX because setting a low system volume doesn't prevent output values greater than +- 1 (which can happen by mistake, e.g. when sending a negative coefficient to a filter) to sound absolutely loud, since CoreAudio applies only a relative attenuation. Values are clipped only before being written to hardware output busses, so although this affects inter-app routing as well (since it would use those same busses), it doesn't affect recording, which still benefits from the full 32bit float range without any clipping.
```

Comment:
here are a few small suggestions:

```suggestion
A Float indicating a safety threshold for output values to be clipped to. This is necessary on macOS because setting a low system volume doesn't prevent output values greater than +/- 1 (which can happen by mistake, e.g. when sending a negative coefficient to a filter) to sound extremely loud, since CoreAudio applies only a relative attenuation. With this threshold, values are clipped just before being written to hardware output busses, so although this affects routing to other apps, it doesn't affect recording, which still benefits from the full 32-bit float range without any clipping.
```

Reply:
Thanks!!